terraform/workspace-variables/production.tfvars.json

```diff
@@ -23,5 +23,8 @@
       "check_rate": 60,
       "contact_group": [151103]
     }
-  }
+  },
+  "azure_tempdata_storage_account_name": "s121p01registerprodtmp",
```

Comment:
Use `pd` rather than `prod` as abbreviation for Production

Reply:
@saliceti These resources already exist. If we are keen, we could delete them and align them with the naming?